### ISSUE TITLE
Remove old non-production S3 buckets

### ISSUE THREAD

**Issue by kelynch** [pulibrary/princeton_ansible#3152]
Depends on new production service being launched and old service being shut down (tickets forthcoming).

## Buckets to ultimately be removed
* `globus-testbucket-20201001`
* `prds-dataspace-staging`

There is an additional bucket that is **CURRENTLY STORING PRODUCTION DATA** which will later need attention paid to it.  A separate ticket should later be made.

blocked by https://github.com/pulibrary/princeton_ansible/issues/3155

## References

* [State of current Globus infrastructure](https://docs.google.com/document/d/1yNokNlCpuufeYG6YTKNKm1phSe7rpo4-BUtKABY1G78/edit)
